modules/custom/az_paragraphs/az_paragraphs_cards/az_paragraphs_cards.install

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file
+ * Az_paragraphs_cards.install.
+ *
+ * Install, update and uninstall functions for az_paragraphs_cards install
+ * profile.
+ */
+
+use Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * Update card deck breakpoints for tablet and desktop media queries.
+ */
+function az_paragraphs_cards_update_9201() {
+
+  if (\Drupal::service('module_handler')->moduleExists('az_paragraphs')) {
+
+    if (!isset($sandbox['total'])) {
+      $count = \Drupal::entityQuery('paragraph')
+        ->condition('type', 'az_cards')
```

Comment:
```suggestion
```
Really any paragraph types can have this behavior enabled on it.

Trying to figure out which ones do, might be complicated, so I think we should just get rid of the condition instead.